.gitignore

```diff
@@ -21,3 +21,6 @@ tmp/
 .tmp/
 temp/
 *.tmp
+
+# zlint user defined rule build cache
+.zlint
```

Comment:
Can these go in `.zig-cache`?

Reply:
yes potentially. I was thinking about it but I think redesigning it to use the user's build system will do that automatically